react-charts/area-chart/step1.md

```diff
@@ -1,10 +1,20 @@
-PatternFly React charts are based on the [Victory chart](https://formidable.com/open-source/victory/docs/victory-chart/) library, along with additional functionality, custom components, and theming for PatternFly. 
-This provides a collection of React based components you can use to build PatternFly patterns with consistent markup, styling, and behavior. 
+Begin by adding a CSS selector to define the chart size.
 
-PatternFly React charts are composed using separate components, so features are more easily customizable. 
-In this course, we're going to build a PatternFly area chart together - starting with a simple chart, adding multiple datasets, tooltips, axis labels, a legend, and concluding by changing the theme color.
-You'll learn how to use PatternFly React chart components together to build a consistent user experience.
+Note: Katacoda is setting up a new React application. You can use the code once the server starts and you see "Welcome to PatternFly" on the lower pane.

```

Comment:
I would go for the recommendation of avoid using you/we and rephrase it a little bit. Of course I'm not sure if this is grammatically sound.
Note: Katacoda is setting up a new React application. It will be ready to code once the server starts and  "Welcome to PatternFly" is displayed on the lower pane.

Reply:
Also, later we use `<strong>Note: </strong>` Should it be the same here?

Reply:
Removing note altogether here